install/image-references

```diff
@@ -26,3 +26,15 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-azure-machine-controllers:v4.0.0
+  - name: baremetal-operator
+    from:
+      kind: DockerImage
+      name: quay.io/metal3-io/baremetal-operator:master
+   - name: baremetal-provider
+    from:
+      kind: DockerImage
+      name: quay.io/metalkube/metalkube-ironic
+   - name: baremetal-inspector
```

Comment:
On the configmap this is called `baremetalIronicInspector` can we use either `baremetalIronicInspector` or baremetalInspector` consistently all around?

Reply:
Changed this to baremetal-ironic-inspector.